client/app/src/main/java/com/microsoft/research/karya/ui/base/BaseActivity.kt

```diff
@@ -257,56 +259,56 @@ abstract class BaseActivity(
-                            window.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
-                            if (onCompletionListener != null) {
-                                onCompletionListener()
-                            }
-                        }
-
-                        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
-                        if (uiCue != null) uiCue()
-                        with(assistantPlayer) {
-                            reset()
-                            setDataSource(audioFilePath)
-                            prepare()
-                            start()
-                        }
-                    }
-                }
-            } else if (onCompletionListener != null) {
-                onCompletionListener()
-            }
-        }
+//        /** If player is not initialized, return */
+//        if (!::assistantPlayer.isInitialized || !assistantAvailable) {
+//            return
+//        }
+//
+//        /** Get the audio file path for the given file */
+//        val name = getString(resId)
+//
+//        ioScope.launch {
+//            var audioLanguageId = languageId
+//            if (languageId == null) {
+//                setAppLanguageJob.join()
+//                audioLanguageId = appLanguageId
+//            }
+//
+//            val resourceId = karyaDb.languageResourceDaoExtra().getIdFromName(name)
+//            val audioFilePath = getBlobPath(
+//                KaryaFileContainer.LANG_RES,
+//                resourceId.toString(),
+//                audioLanguageId.toString()
+//            )
+//
+//            /** Play the recording if file exists. Else silently fail */
+//            if (File(audioFilePath).exists()) {
+//                if (assistantAvailable) {
+//                    uiScope.launch {
+//                        if (assistantPlayer.isPlaying) assistantPlayer.stop()
+//
+//                        /** Set on completion listener if one is provided */
+//                        assistantPlayer.setOnCompletionListener {
+//                            window.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
+//                            if (onCompletionListener != null) {
+//                                onCompletionListener()
+//                            }
+//                        }
+//
+//                        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
+//                        if (uiCue != null) uiCue()
+//                        with(assistantPlayer) {
+//                            reset()
+//                            setDataSource(audioFilePath)
+//                            prepare()
+//                            start()
+//                        }
+//                    }
+//                }
+//            } else if (onCompletionListener != null) {
+//                onCompletionListener()
+//            }
+//        }
```

Comment:
I don't see any benefit in commenting it. Can you please explain why we are doing it?

Reply:
We do not have files for loading assistant audio files. This makes the app not crash since we are not downloading those files